TutorLizard.BusinessLogic/Models/DTOs/Requests/CreateAdRequestRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace TutorLizard.BusinessLogic.Models.DTOs.Requests;
+public class CreateAdRequestRequest
+{
+    public int StudentId { get; set; }
+
+    [Required]
+    public int AdId { get; set; }
+
+    [DisplayName("Wiadomość dla nauczyciela")]
+    [Required(ErrorMessage = "To pole jest wymagane")]
+    [MaxLength(500, ErrorMessage = "Maksymalna długość: 500 znaków")]
+    public string Message { get; set; }
+
+    [DisplayName("Nauczanie zdalne")]
+    [Required(ErrorMessage = "To pole jest wymagane")]
```

Comment:
if you force to check isRemote it's no longer optional. You should be able to choose if you prefer to take lessons remotely or locally.

Reply:
It's required for a value to be set, but it can still be false. If I click button without touching this checkbox, `ModelState` is still valid and the value is `false`.
Honestly, this attribute changes nothing. I thought maybe that maybe `ModelState` would be invalid if the form didn't include the checkbox at all, but it's just set to default value and passes through without an issue. I'll remove it.